src/GovUk.Education.ExploreEducationStatistics.Publisher/Services/ReleaseInfoService.cs

```diff
@@ -0,0 +1,87 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using GovUk.Education.ExploreEducationStatistics.Common.Services.Interfaces;
+using GovUk.Education.ExploreEducationStatistics.Publisher.Model;
+using GovUk.Education.ExploreEducationStatistics.Publisher.Services.Interfaces;
+using Microsoft.Azure.Cosmos.Table;
+
+namespace GovUk.Education.ExploreEducationStatistics.Publisher.Services
+{
+    public class ReleaseInfoService : IReleaseInfoService
```

Comment:
Would be good to get a comment at the top here to explain the function of this class

Reply:
It's been renamed the ReleaseStatus service now and lots of renaming of the methods. Hopefully by looking at the methods of IReleaseStatusService it's pretty clear what's going on now.